test/pkg/steps/maven.go

```diff
@@ -71,7 +76,9 @@ func (data *Data) localServiceBuiltByMavenWithProfileAndOptions(serviceName stri
 	if mavenConfig.Native {
 		mvnCmd = mvnCmd.Profiles(nativeProfile)
 	}
-	output, err := mvnCmd.Execute("clean", "package")
-	framework.GetLogger(data.Namespace).Debug(output)
```

Comment:
Please don't remove this debug entry, can come handy.

Reply:
sure, will revert